OamLog.yaml

```diff
@@ -519,33 +581,33 @@ paths:
                       type: object
                       required:
                         - application-name
```

Comment:
line 598 :: PromptForBequeathingDataCausesRequestForBroadcastingInfoAboutServerReplacement :: old-release-number :: description ::
Can we update the release-number description to be pointing to the http-server/release-number instead of OldRelease/release-number, because OldRelease will contain information of already outdated release of this application. And our intention is to request RO to broadcast about replacement of current release (which is expected to be deregistered - http-server) with new release of this application

Reply:
Thanks for updating the path.

Also, the first line of description shall be changed to "Own release number "

Reply:
Changed as proposed.